integration_tests/saiutil/saiutil.go

```diff
@@ -103,3 +131,138 @@ func CreateNeighbor(t testing.TB, dut *ondatra.DUTDevice, ip string, dmac string
 		t.Fatal(err)
 	}
 }
+
+type InterfaceRef struct {
```

Comment:
Any reason not to use NetworkInstance_Afts_NextHop_InterfaceRef?

Reply:
there are other InterfaceRef types in the OC, this way we always use the same type